RobotFramework_DoIP/DoipKeywords.py

```diff
@@ -556,4 +556,42 @@ def request_diagnostic_power_mode(self):
             resp = self.client.request_diagnostic_power_mode()
             logger.info(f"diagnostic_power_mode: {resp.diagnostic_power_mode}")
         else:
-            logger.warn(f"No active DoIP connection. Unable to request diagnostic power mode.")
\ No newline at end of file
+            logger.warn(f"No active DoIP connection. Unable to request diagnostic power mode.")
+
+    @keyword("Build payload")
+    def build_payload(self, request):
+        """
+        **Description:**
+
+            Build payload
+
+        **Parameters:**
+
+            * param ``request`` (required): hex data
+            * type ``request``: hex
+
+        **Return:**
+
+            return bytes data of the request
+
+        **Exception:**
+
+            * raise ValueError: if request is None
+
+        **Usage:**
+
+            * Build payload by protocol version | ${request}
+            * Build payload by protocol version | hex_value
+        """
+
+        if request is None:
+            raise ValueError("The request cannot be None.")
+
+        msg = bytes.fromhex(request)
+        from doipclient import messages
```

Comment:
Hi Tri, 
why do you not import messages in the top of the file, e.g. below line 13?
Is there a reason to do this here?
Thank you,
 Thomas

Reply:
Hi Thomas,
There wasn’t a specific reason; I simply forgot to move it to the top.
Thanks for your finding.
Tri